Targets/Apps/ZScaler.tkape

```diff
@@ -0,0 +1,15 @@
+Description: Zscaler Logs
+Author: Vito Alfano
+Version: 1.0
+Id: 0f3c8e4c-2307-4dcf-846b-17a888a585a6
+RecreateDirectories: true
+Targets:
+    -
+        Name: Zscaler Logs
+        Category: ApplicationLogs
+        Path: C:\Users\%user%\AppData\Local\Zscaler
+        FileMask: '*'
+        Comment: "Contains Zscaler agent Logs entries"
+
+# Documentation
+#
```

Comment:
Need to add N/A here and then a new line after

Reply:
I can't get to it right now but I'll take care of it if you haven't already by the time I get to my computer